### DISCUSSION TITLE
Display a popup toast message when we press the keyboard shortcut for a certain action

### DISCUSSION THREAD

**Discussion by Psybertr0n** [f1multiviewer/issue-tracker#228] [Feature requests]
At present we have several keyboard shortcuts that we use to do things like sync all players to one player, pan audio in OBC streams, increase or decrease latency for a certain player. Most of these actions require us to pull up the debug menu though. 

Instead of bringing up the debug menu to see our change take effect, it would be nice to get a brief popup message within the bounds of the player window to give confirmation of the action we just triggered with the keyboard shortcut. This would be something similar to toast messages we have in Android as pictured in the GIF below.

![63286541](https://user-images.githubusercontent.com/16277629/193308679-01b37462-2d7b-417d-a60a-3faf9e7a4dab.gif)

**Some examples I thought of are as follows:**

1. While watching a session replay when we click on the main feed player and press "s" to sync all players to it, we could get toast messages on all other players saying "Synced to International" or "Synced to F1 Live". Similarly if we click on the Data Channel stream and press "s" all other players should show the toast message "Synced to Data Channel".
2. When we pan audio for an OBC stream, that OBC player window could show the toast message "Audio panned to XYZ", XYZ being Center or Left or Right.
3. While watching a live session if we want to change the latency of a certain stream by pressing arrow keys or shift+arrow keys, then that player window could show the toast message "Target latency increased by n second and is now m seconds". Example: "Target latency increased by 0.1 second and is now 37.7 seconds".

**Advantages of doing this:**

1. Not having to bring up the debug menu and instead getting toast messages is more convenient because there wouldn't be anything large occluding the content of the stream.
2. You also wouldn't need to manually press more keys to show and hide the debug menu and you can just focus on your stream.
3. Having these visual acknowledgements would be more beginner friendly as it gives the user a quick and precise visual confirmation of the specific action they took rather than searching for it in the debug menu.
4. Having visual indications like this would bring UX in line with what people have experienced with other native video players like VLC or mpv. I have shown the screen recording of a video playing within mpv player below where you can see visual indications of the keyboard shortcuts I press in the top left corner.

https://user-images.githubusercontent.com/16277629/193317857-dee2f503-0494-43ec-a999-7590676c006a.mp4

**Comment by f1multiviewer** [comment]
This is a great idea! I'll see what I can do!

**Reply by Psybertr0n** [reply]
Hey Multiviewer, did you get a chance to explore this? I feel this would be a very welcome QoL change.
